### ISSUE TITLE
[Item] Raptor Hide Belt unavailable to horde characters

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#4618]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

I bought the Raptor Hide Belt Leatherworking pattern from Alliance NPC Androd Fadran and proceded to transfer it to Horde through the neutral action house. This pattern is an Alliance exclusive, with the Horde version being Raptor Hide Harness. It is currently not possible to learn the pattern, instead I get the message "You can never use that item". The pattern does not transform into the Horde version when you pick it up from a mail box.

**Expected behaviour**: Tell us what should happen instead

The 100% correct way is a little bit unclear but from what I can gather, the pattern should transform into the Horde version (Raptor Hide Harness) IF a Leatherworker picks up this pattern (Raptor Hide Belt) if he doesn't already have the Horde version of the pattern. If he does have Raptor Hide Harness, the pattern should NOT transform and he should be able to learn it. It seems like the retail way of going around this transformation, was to have somebody without Leatherworking, pick it up from the neutral auction house. 

**Steps to reproduce the problem**:

1. Have Leatherworking with a skill of 165.
2. Buy the Alliance version of the pattern and transfer it to Horde side.
3. Try to learn the pattern.

**Include proofs for this behaviour**
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Alliance pattern: http://www.wowhead.com/item=13288/pattern-raptor-hide-belt#sold-by
Horde pattern: http://www.wowhead.com/item=13287/pattern-raptor-hide-harness
Alliance NPC: http://www.wowhead.com/npc=2816/androd-fadran

Expected behavior taken from the comments section of the Alliance pattern:

"Confirmed that Horde can have this pattern without it changing to the harness pattern. You MUST have the raptor hide harness pattern learned first before putting this pattern in your inventory or it will change to the Horde-only harness pattern."

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Alliance pattern ID:  13288
Horde pattern ID  :  13287
Alliance NPC ID    :  2816

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

Screenshots of the Alliance version of the pattern and current behavior:
https://puu.sh/uSOhp/510fb48ec6.png
https://puu.sh/uSOlM/aacbf508f5.png

**Comment by Rushor** [comment]
DB/Item: Raptor Hide Belt + Pattern: Raptor Hide Harness - Can now also be used with a alliance/horde character

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4618
* commit: e3bba0f7462bbedbe5e89698009ccab2ee033593

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
